README.md

```diff
@@ -329,10 +330,12 @@ function()
   vim.lsp.buf.formatting_sync(nil, 1500)
 end
 ```
-
-If you need to call a function from a plugin, but do not want the require that the plugin be
-loaded at the time you define your keymaps (like if you're using Packer to lazy-load plugins),
-you can use the `lazy_required_fn` helper:
+<!-- don't like the wording 'don't require the plugin _be_ loaded
```

Comment:
how about "do not want to require that the plugin is loaded at the time you define your keymaps"